<?xml version="1.0" ?>
<launch>
	<node name="StaticGrasp" pkg="robot_task" type="static_grasp" output="screen" >

      <param name="gripper_width" value = "10" />
      <param name="camera_x" value = "0.435" />
      <param name="camera_y" value = "0.165" />
      <param name="camera_z" value = "0.031" />

      <param name="file_path" value = "/home/panda/" />
	</node>


</launch>